src/client/components/templates/district-office/district-office.jsx

```diff
@@ -61,6 +76,33 @@ class DistrictOfficeTemplate extends React.Component {
   }
 }
 
+const Leadership = ({ items }) => {
```

Comment:
```suggestion
const Leadership = ({ leaders }) => {
```
We could change this to "leaders" to be a little more descriptive since we have other components on this page taking in "items"

Reply:
I like to use items within the component because it is a generic description of functionality. Outside of the component I prefer to call the object leaders beause it is more descriptive of the data outside of the context of the component.